day_7/10_little_monkeys.rb

```diff
@@ -0,0 +1,27 @@
+
+monkeys = [
+  'Ten',
+  'Nine',
+  'Eight',
+  'Seven',
+  'Six',
+  'Five',
+  'Four',
+  'Three',
+  'Two',
+  'One'
+]
+
+monkeys.each do |monkeys|
```

Comment:
@garretwolf - Nice work here! One suggestion is to change the placeholder name of your current element in the iteration to something _singular_ and _different_ from the name of the array, ie. line 15 could become
`monkeys.each do |monkey|`

Keep it up!